catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -203,6 +219,33 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
         outer.validators = validators;
     }
 
+    @SuppressWarnings("unchecked")
+    private void parseMetacardValidators(Map<String, Object> root, Outer outer) {
+        if (root == null || root.get(METACARD_VALIDATORS_PROPERTY) == null) {
+            return;
+        }
+
+        List<MetacardValidatorDefinition> metacardValidators = new ArrayList<>();
+        Object metacardValidatorsObj = root.get(METACARD_VALIDATORS_PROPERTY);
+        List<Map<String, Object>> metacardValidatorsList =
```

Comment:
Do you know that this cast will always succeed? Do we need type checking for the list, the map and the map contents?